pkg/fakerp/fakerp.go

```diff
@@ -200,7 +200,12 @@ func createOrUpdateWrapper(ctx context.Context, p api.Plugin, log *logrus.Entry,
 		return nil, err
 	}
 
+	if !isUpdate {
+		cs.Properties.MonitorProfile.Enabled = true
```

Comment:
I thought we were not going to turn this on by default, isn't quite expensive?

Reply:
I think its a good idea to have this always on in development so we are confident that the monitoring integration works (eating our own dog food). There are only a few dev clusters being created each of which lasts a maximum of 3 days. Our workspaces currently have a retention period of 31 days which is included in our pricing plan.

Reply:
this should not be here - update the create.yaml manifest.